CHANGELOG.md

```diff
@@ -14,6 +14,7 @@ This project adheres to [Semantic Versioning](https://semver.org/).
 ### Changed
 - Remove the obsolete core classes (#267)
 - Adopt more of the default Symfony project structure (#265, #268, #269, #270)
+- Change composer configuration to reflect new repository name 'core'
```

Comment:
This needs to go to the top of the changes list (as it's ordered "newest first").